crates/stark-backend/tests/interaction/mod.rs

```diff
@@ -118,9 +116,6 @@ fn test_interaction_stark_multi_rows_neg() {
         2,
     );
     let receiver_air = DummyInteractionAir::new(1, false, 0);
-    USE_DEBUG_BUILDER.with(|debug| {
-        *debug.lock().unwrap() = false;
-    });
     let res = verify_interactions(
```

Comment:
No disable here?

Reply:
`verify_interactions` doesn't use `test`, only `prover` so now that doesn't include the debug builder